pkg/server/cloud/aws/instances.go

```diff
@@ -44,6 +44,13 @@ const (
 	AvailableWaitTimeout      = 30 * time.Second
 )
 
+type EbsSpecs struct {
+	Iops       int64
+	Throughput int64
```

Comment:
We should change these two to be pointers, since in some cases, they are optional (see below).